cms/templates/js/active-video-upload-list.underscore

```diff
@@ -6,5 +6,5 @@
 </form>
 <section class="active-video-upload-container">
     <h3 class="sr"><%= gettext("Active Uploads") %></h3>
-    <ul class="active-video-upload-list"></ul>
+    <ul class="active-video-upload-list" aria-live="polite"></ul>
```

Comment:
@cptvitamin CC @tanmaykm 

We'll remove those `aria-live="polite"` attributes if they're not helping.  I've never been able to get VoiceOver to recognize them, so my apologies for adding them in blind.

However, I'm wondering if there's a better way to send the status update message on "Upload complete"?  Currently, if there's an upload error, we leave the video upload item and its progress bar on screen, with the status marked as "Upload error".

However, with successful uploads, we remove the video upload item from the active video uploads list, and expect its absence to indicate successful upload.

Wouldn't it be better to leave them all on screen, so that the status and progress can always be seen?  That way, we can set an `aria-live="polite"` attribute on the `p.video-detail-status` message, and everyone gets the benefits of seeing the status messages.

I think it's fine to update the table as well, but I believe this would take care of the notification requirement.

e.g.
![screen shot 2016-10-19 at 2 56 09 pm](https://cloud.githubusercontent.com/assets/7556571/19505724/cdefed24-960c-11e6-96cb-12490ba66214.png)